pom.xml

```diff
@@ -101,6 +112,25 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.8</version>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <version>${findbugs-maven-plugin.version}</version>
+        <configuration>
+          <xmlOutput>true</xmlOutput>
+          <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
+          <failOnError>${findbugs-maven-plugin.failOnError}</failOnError>
```

Comment:
i think @oleg-nenashev followed existing style with controlling jenkins build processes with properties that come from 'pom' project. Handling env properties from profile also makes sense but seems doesn't fit idea with enforcing it.